lib/Client.test.ts

```diff
@@ -257,6 +259,7 @@ describe("Client", () => {
         new HttpHeaders({
           "x-api-key": "key-secret",
           "content-type": "application/json",
+          "user-agent": "journyio-js-sdk/0.0.0",
```

Comment:
```suggestion
          "user-agent": "js-sdk/0.0.0",
```

What do you think about shorting this?

Reply:
Good idea, I do not think someone else will create a js-sdk for our API since we have created one... :see_no_evil: